src/skeleton/skeleton.js

```diff
@@ -0,0 +1,33 @@
+
+import * as React from 'react';
+import { getOverrides } from '../helpers/overrides.js';
+import type {SkeletonPropsT} from './types.js';
+import {
+  StyledRoot,
+  StyledRow,
+} from './styled-components.js';
+
+
+class Skeleton extends React.Component<SkeletonPropsT> {
+  render() {
+    const { overrides = {} } = this.props;
+    const [Root, rootProps] = getOverrides(overrides.Root, StyledRoot);
+    const [Row, rowProps] = getOverrides(overrides.Row, StyledRow);
+
+    if (typeof this.props.rows === 'number') {
+      return (
+        <Root {...rootProps}>
```

Comment:
It may get complex to render root differently when rows are present or not. Maybe this should be a new override called RowsContainer?